MANIFEST.in

```diff
@@ -1,2 +1,2 @@
-semeio/jobs/config_jobs
-semeio/jobs/config_workflow_jobs
+include semeio/jobs/config_jobs/*
```

Comment:
Does this work for potential future subdirectories in `config_jobs`? It might be that either `semeio/jobs/config_jobs` or `semeio/jobs/config_jobs/**/*` is more correct.

Reply:
> The patterns here are glob-style patterns: * matches zero or more regular filename characters

https://packaging.python.org/guides/using-manifest-in/

Soooo, yes?

Reply:
No wait, you are correct! 

> Setuptools also has undocumented support for ** matching zero or more characters including forward slash, backslash, and colon.